stl/inc/limits

```diff
@@ -1013,20 +1015,83 @@ public:
 };
 
 #ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+
+inline bool _Is_tzcnt_supported() {
+    // tzcnt and lzcnt support are indicated with
+    // different cpuid flags.
+    int cpuinfo[4];
+    __cpuid(cpuinfo, 0x07);
+    return cpuinfo[1] & (1 << 3);
+}
+
+
+// see "Hacker's Delight" section 5-3
+template <typename _Ty>
+constexpr int _Countl_zero_helper(_Ty _Val) {

```

Comment:
Given that these helpers are basically `constexpr` fallbacks it might be more expressive to name them `meow_constexpr` instead of `meow_helper`